### PR TITLE
feat: onboarding graphics and ui update

### PR DESCRIPTION
**Pull request by jhasuraj01**
image

image

image

image

image

image

image

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6306 On cysync graphics are missing for device and cards](https://app.clickup.com/t/9002019994/PRF-6306)

File: packages/cysync-core/src/components/AppUpdateFailedFallback.tsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
   AlertBox,
-  AppUpdateIcon,
   Container,
   DialogBox,
   DialogBoxBody,
   LangDisplay,
   CopyContainer,
   Typography,
-  useTheme,
+  CySyncDownloadRed,
```
Comment (irshadCypherock) [inline packages/cysync-core/src/components/AppUpdateFailedFallback.tsx]:
```suggestion
  CySyncDownloadRedIcon,
```

File: packages/cysync-core/src/pages/OnBoarding/AppUpdate/Dialogs/index.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import {
-  AppUpdateIcon,
   ConfirmationDialog,
   ErrorDialog,
   ProgressDialog,
   SuccessDialog,
+  CySyncDownloadGreen,
```
Comment (irshadCypherock) [inline packages/cysync-core/src/pages/OnBoarding/AppUpdate/Dialogs/index.tsx]:
```suggestion
  CySyncDownloadGreenIcon,
```

File: packages/ui/src/components/molecules/Dialog/ErrorDialog.tsx [hunk 3]
```diff
@@ -3,11 +3,16 @@ import React, { ReactNode } from 'react';
 import { DialogBoxProps } from './DialogBox';
 import { IconDialogBox } from './IconDialogBox';
 
-import { ServerErrorIcon, FailIcon, SettingsWrongIcon } from '../../../assets';
+import {
+  ServerErrorIcon,
+  FailIcon,
+  SettingsWrongIcon,
+  CySyncDownloadRed,
```
Comment (irshadCypherock) [inline packages/ui/src/components/molecules/Dialog/ErrorDialog.tsx]:
```suggestion
  CySyncDownloadRedIcon,
```

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/molecules/Dialog/ErrorDialog.tsx (hunk 3 above)
Comment (jhasuraj01) [reply packages/ui/src/components/molecules/Dialog/ErrorDialog.tsx]: Refactoring Done

File: packages/cysync-core/src/pages/OnBoarding/AppUpdate/Dialogs/index.tsx (hunk 2 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/pages/OnBoarding/AppUpdate/Dialogs/index.tsx]: Refactoring Done

File: packages/cysync-core/src/components/AppUpdateFailedFallback.tsx (hunk 1 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/components/AppUpdateFailedFallback.tsx]: Refactoring Done

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**